ibis/backends/sql/datatypes.py

```diff
@@ -27,8 +27,6 @@
     typecode.ENUM16: dt.String,
     typecode.FLOAT: dt.Float32,
     typecode.FIXEDSTRING: dt.String,
-    typecode.GEOMETRY: partial(dt.GeoSpatial, geotype="geometry"),
```

Comment:
Small question, we don't need these ones any more because we will always git this path
`method := getattr(cls, f"_from_sqlglot_{typecode.name}", None)` in `to_ibis()` now?

Reply:
No, they were never used prior to this PR as well.